packages/components/src/Combobox/Combobox.module.css

```diff
@@ -10,4 +10,9 @@
 
 .wrapper {
   display: inline-block;
+  max-width: 100%;
+}
+
+.wrapper > [role="combobox"] {
```

Comment:
I'd prefer to not have to also apply this to the child. if we could set it solely on the parent wrapper, that would be great.